File: pset6/mario/mario.py
from cs50 import get_int

height = get_int("Height: ")

# Input validation
while height not in range(1, 9):
    height = int(input("Height: "))

# Looping through each row
for i in range(1, height+1):
    # Print the spaces padded in front
    for j in range(0, height-i):
        print(" ", end='')

    # Print the hashtags
    for j in range(0, i):
        print("#", end='')

    # Print the 2 spaces in between hashtags
    print("  ", end='')

    # Print the hashtags
    for j in range(0, i):
        print("#", end='')

    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    // Looping through each row
    for (int i = 1; i <= height; i++)
    {
        // Print the spaces padded in front
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }

        // Print the hashtags
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Print the 2 spaces in between hashtags
        printf("  ");

        // Print the hashtags
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Print the newline
        printf("\n");
    }
}